evalml/tests/pipeline_tests/test_component_graph.py

```diff
@@ -1531,31 +1412,6 @@ def test_component_graph_sampler_y_passes(mock_estimator_fit):
     assert len(mock_estimator_fit.call_args[0][0]) == int(1.25 * 90)
 
 
-@patch("evalml.pipelines.components.estimators.RandomForestClassifier.fit")
-@patch("evalml.pipelines.components.estimators.DecisionTreeClassifier.fit")
-def test_component_graph_sampler_same_given_components(mock_dt_fit, mock_rf_fit):
```

Comment:
I'm not sure what the original purpose of this test was, just verifying that `SMOTE Oversampler` was included in both graphs?

Reply:
I think it was just testing that with different estimators that the oversampler was the same and received the same input :)